test/martha/martha_v3.test.js

```diff
@@ -670,12 +637,12 @@ test.serial('martha_v3 parses BDC response correctly', async (t) => {
     );
     t.is(response.statusCode, 200);
     const result = response.send.lastCall.args[0];
-    // Bond was called twice to get SA key and access token.
-    // DRS server was called twice to get DRS response and signed URL.
-    // 2 + 2 = 4.
-    sinon.assert.callCount(getJsonFromApiStub, 4);
+    // Bond was called once to get SA key.
+    // DRS server was called once to get DRS response.
+    // 1 + 1 = 2.
```

Comment:
Now that you're a certified Martha expert, what are your thoughts on these types of comments existing _only_ in this one test in the middle of the test suite?

Reply:
this can probably go but since the whole PR will be reverted they may sneak back in 🙂